en/docs/fhir/guides/exposing-an-api.md

```
@@ -55,25 +55,25 @@ If you have already built your FHIR integration service, you can follow the step
 
 1. Sign in to Healthcare Developer Portal.
 
-    ![WSO2 Open Healthcare Developer Portal](/assets/img/guildes/exposing-an-api/oh-developer-portal.png)
+    ![WSO2 Open Healthcare Developer Portal](/assets/img/guides/exposing-an-api/oh-developer-portal.png)
 
 1. Navigate to Applications tab and to create an application, click on **Add New Application** button
     1. An application is a logical representation of a physical application such as a mobile app, web app, device, etc. An API subscription is created, authenticated, and managed through an application. Find out more about applications by clicking [here](https://apim.docs.wso2.com/en/4.1.0/consume/manage-application/create-application]).
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)